### ISSUE TITLE
run_docker.py failure caused by module not found error

### ISSUE THREAD

**Issue by jygt2022** [deepmind/alphafold#369]
Hi AlphaFold team,

When I run the run_docker.py with the following command,

```bash
python3 docker/run_docker.py \
  --fasta_paths=/home/ubuntu/workspace/fasta_files/query.fasta \
  --max_template_date=2020-05-14 \
  --data_dir=/disk1/genetic_database/
```

I got this error message:

```
I0215 05:53:50.543966 140037725366080 run_docker.py:107] Mounting /home/ubuntu/workspace/fasta_files -> /mnt/fasta_path_0
I0215 05:53:50.544103 140037725366080 run_docker.py:107] Mounting /disk1/genetic_database/uniref90 -> /mnt/uniref90_database_path
I0215 05:53:50.544165 140037725366080 run_docker.py:107] Mounting /disk1/genetic_database/mgnify -> /mnt/mgnify_database_path
I0215 05:53:50.544217 140037725366080 run_docker.py:107] Mounting /disk1 -> /mnt/data_dir
I0215 05:53:50.544266 140037725366080 run_docker.py:107] Mounting /disk1/genetic_database/pdb_mmcif -> /mnt/template_mmcif_dir
I0215 05:53:50.544313 140037725366080 run_docker.py:107] Mounting /disk1/genetic_database/pdb_mmcif -> /mnt/obsolete_pdbs_path
I0215 05:53:50.544359 140037725366080 run_docker.py:107] Mounting /disk1/genetic_database/pdb70 -> /mnt/pdb70_database_path
I0215 05:53:50.544407 140037725366080 run_docker.py:107] Mounting /disk1/genetic_database/uniclust30/uniclust30_2018_08 -> /mnt/uniclust30_database_path
I0215 05:53:50.544457 140037725366080 run_docker.py:107] Mounting /disk1/genetic_database/bfd -> /mnt/bfd_database_path
I0215 05:53:51.331728 140037725366080 run_docker.py:247] Traceback (most recent call last):
I0215 05:53:51.331906 140037725366080 run_docker.py:247] File "/app/alphafold/run_alphafold.py", line 29, in <module>
I0215 05:53:51.331975 140037725366080 run_docker.py:247] from alphafold.common import protein
I0215 05:53:51.332035 140037725366080 run_docker.py:247] ModuleNotFoundError: No module named 'alphafold.common'
```

I am new to AlphaFold and I followed each step from the README to 

1. Configure Nvidia GPU.
2. Download the genetic database files.
3. Build the Docker image.
4. Pip3 install the requirements.
5. Configure output directory.
6. Run the run_docker.py command.

Everything works correctly until I hit the error at step 6. Could you please advise? Thanks!

**Comment by Augustin-Zidek** [comment]
This looks like a problem with the cloned alphafold repository. Could you try to re-clone it, re-build the Docker image and run again?

**Comment by jygt2022** [comment]
Hi Augustin-Zidek,

Thanks for your response. I re-clone the alphafold repo, re-build the Docker images and run it again, but I still get the same error.

`
I0216 18:20:13.442532 140222277302080 run_docker.py:247] Traceback (most recent call last):
I0216 18:20:13.442734 140222277302080 run_docker.py:247] File "/app/alphafold/run_alphafold.py", line 29, in <module>
I0216 18:20:13.442825 140222277302080 run_docker.py:247] from alphafold.common import protein
I0216 18:20:13.442888 140222277302080 run_docker.py:247] ModuleNotFoundError: No module named 'alphafold.common'
`

**Comment by Augustin-Zidek** [comment]
Could you check if there are `__init__.py` files in `alphafold/` and all its subdirectories?

**Comment by jygt2022** [comment]
Hi Augustin-Zidek,

We have resolved this issue! We applied `chomd ` to change the permission of all files and sub-directories under `alphafold/`. In our system, the `alphafold/` belongs to the default user "root" while the user that runs alphafold is "ubuntu" that is why the module is not found. I suggest to add a `chmod` command in the docker file to update file permissions to avoid this potential risk. Thanks for your advice in the troubleshooting!
